packages/pwa-buildpack/lib/Utilities/graphQL.js

```diff
@@ -0,0 +1,63 @@
+const fetch = require('node-fetch');
```

Comment:
So GitHub didn't pick it up, but I renamed `fetcherUtils` to `graphQL` (and their associated test files).

This is instrumentation to make GraphQL calls so I think this name is more indicative of what this module's job is.